zwave_js_server/model/node.py

```diff
@@ -246,6 +271,19 @@ async def async_set_value(self, val: Union[Value, str], new_value: Any) -> bool:
         # a value may be specified as value_id or the value itself
         if not isinstance(val, Value):
             val = self.values[val]
+
+        if not val.metadata.writeable:
+            raise TypeError("This configuration value is read only")
```

Comment:
If the exception is library user facing the exception should inherit from the library base exception.

If it's just for library developers, we don't need to be as stringent.

Reply:
I'm not sure at this point, so safer to inherit from the library base exception